src/routes/user/validation.js

```diff
@@ -19,9 +17,8 @@ const validateField = {
     'string.empty': `Email must contain value`,
     'any.required': `Email is a required field`,
   }),
-  password: password.password,
-  newPassword: password.password,
-  newPasswordRepeat: password.password,
+  password: password,
+  newPassword: password,
```

Comment:
With the changes to the code you introduced, I think `validateField` could hold just the key `password` so you could:
1) move the Joi object which the variable `password` holds directly to `validateField` and delete the variable,
2) remove the key `newPassword` from `validateField`
3) use `password` validation for the field `newPassword` in `passwdEditValidation`: `newPassword: validateField.password`